### ISSUE TITLE
Feature: show toast on new song

### ISSUE THREAD

**Issue by simonemazzoni** [kmikiy/SpotMenu#82]
Hello,
first of all congrats for the good work.

I want to propose to show automatically the toast showed by pressing `cmd + ctrl + m` on song change (And maybe add a toggle for it in preferences menu for those who doesn't want this to happen). I often listen to random playlists proposed by Spotify, and every time a new song starts I have to manually invoke the toast which isn't that handy while working. :)

Thanks
Simone

**Comment by Zazerr** [comment]
Show _toast_? What is this *toast* you speak of??

**Comment by simonemazzoni** [comment]
Hello @Zazerr 
I'm talking of this toast (showed using the shortcut `cmd + ctrl + m` as I said before)
![screen shot 2017-11-30 at 10 40 47 2](https://user-images.githubusercontent.com/3217537/33424149-311b6808-d5bb-11e7-9464-27c93e8e43df.png)

The one that logic is written [here](https://github.com/kmikiy/SpotMenu/blob/master/SpotMenu/Toast/Toast.swift) I guess (I'm not an objective-c expert)

**Comment by kmikiy** [comment]
@simonemazzoni i'll add on option to automatically show the toast when the song changes. I don't know when i'll have time to do it.

I also might add an option to show a **notification** when the song changes. Similar to what iTunes has built in. The type that slides in from the top right.

@Zazerr it's sometimes called HUD. I used toast originally because its similar to a notification on WP7 which was called toast.

**Comment by simonemazzoni** [comment]
@kmikiy thank you so much, this is great news! :)

> I also might add an option to show a notification when the song changes. Similar to what iTunes has built in. The type that slides in from the top right.

That would be great too. Another widget I used in the past does that. (It's called Simplify if you want to have a look)

I'll wait for the update.

**Comment by Zazerr** [comment]
Am I understanding this correctly - when a new song plays, a popup appears in the top right corner which displays the now playing song? I'm a little confused...

**Comment by joshghent** [comment]
@Zazerr Yes that's correct, and then it will disappear after a certain length of time.
https://ux.stackexchange.com/questions/11998/what-is-a-toast-notification
